src/test/emulators/functionsEmulatorRuntime.spec.ts

```diff
@@ -391,47 +391,47 @@ describe("FunctionsEmulator-Runtime", () => {
         expect(info.url).to.eql("http://localhost:9090/");
       }).timeout(TIMEOUT_MED);
 
-      it("should merge .initializeApp arguments from user", async () => {
```

Comment:
Got rid of this wacky test which had been skipped in CI anyway (see lines below) and is no longer useful.